packages/@aws-cdk/aws-autoscaling/test/test.auto-scaling-group.ts

```diff
@@ -748,13 +754,21 @@ export = {
             VolumeSize: 500,
             VolumeType: 'sc1',
           },
-          NoDevice: true,
+          NoDevice: ABSENT,
```

Comment:
Good! This should have probably been an integration test so we could have catch this actually not working